JSDemos/Demos/DataGrid/ToolbarCustomization/description.md

```diff
@@ -1 +1,13 @@
-The **DataGrid** widget allows you to&nbsp;customize its toolbar using the **onToolbarPreparing** function. You can add new items to&nbsp;the toolbar or&nbsp;customize the existing ones. In&nbsp;this demo, the toolbar contains a&nbsp;collection of&nbsp;other widgets from the DevExtreme HTML5&nbsp;UI Widgets library.
\ No newline at end of file
+The toolbar contains predefined and custom controls that execute various commands.
+
+The **DataGrid** uses the DevExtreme [Toolbar](/Demos/WidgetsGallery/Demo/Toolbar/Overview/) component as the toolbar. To customize the toolbar, implement the [onToolbarPreparing](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onToolbarPreparing) function. Its parameter contains the [Toolbar's configuration](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/) in the **toolbarOptions** field. You can access toolbar items via the **toolbarOptions**.[items](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/) array. Modify this array to add or remove toolbar items.
```

Comment:
I'm mostly trying to avoid the repetition of the word 'Toolbar' that appears in almost every sentence, twice in some sentences. 

```suggestion
The **DataGrid** uses an integrated DevExtreme [Toolbar](/Demos/WidgetsGallery/Demo/Toolbar/Overview/) component to display built-in and custom commands. To add or remove toolbar items, handle the [onToolbarPreparing](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onToolbarPreparing) event. Use the parameter's **toolbarOptions** field to access the [Toolbar's configuration](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/) object. This object includes the command collection: the **toolbarOptions**.[items](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/) array. 
```

Reply:
```suggestion
The **DataGrid** uses an integrated DevExtreme [Toolbar](/Demos/WidgetsGallery/Demo/Toolbar/Overview/) component to display predefined and custom commands. To add or remove toolbar items, implement the [onToolbarPreparing](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onToolbarPreparing) function. Use its parameter's **toolbarOptions** field to access the [Toolbar's configuration](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/) object. This object includes the command collection: the **toolbarOptions**.[items](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/) array.
```

1) 'predefined' because it stands for DevExtreme widgets and Predefined controls (e.g. columnChooser).

2) 'implement the onToolbarPreparing function' because onToolbarPreparing is a function.

Is it good?